pify dependency missing in @keyv/sqlite ?

After installing keyv + @keyv/sqlite and adding the basic example from readme i get this error:

Error: Cannot find module 'pify'
Require stack:
- mypath\node_modules\@keyv\sqlite\src\index.js
- mypath\node_modules\keyv\src\index.js
- mypath\server.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at Object.<anonymous> (mypath\node_modules\@keyv\sqlite\src\index.js:5:14)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'mypath\\node_modules\\@keyv\\sqlite\\src\\index.js',
    'mypath\node_modules\\keyv\\src\\index.js',
    'mypath\\server.js'
  ]

Should probably have been added in commit 768014a72e722a0ea0ddae4a185fb16a022e6b08 unless I am missing something?

--- a/packages/sqlite/package.json
+++ b/packages/sqlite/package.json
@@ -42,7 +42,8 @@
 	},
 	"homepage": "https://github.com/jaredwray/keyv",
 	"dependencies": {
-		"sqlite3": "^5.0.3"
+		"sqlite3": "^5.0.3",
+		"pify": "^5.0.0"
 	},
 	"devDependencies": {
 		"@keyv/test-suite": "*",

